Added JS binary search algorithm implementation.

peakercope opened the pull request.

peakercope committed:
Added JS binary search algorithm implementation.

rohitjmathew commented:
@peakercope can you create an issue as mentioned in [CONTRIBUTING.md](https://github.com/Sathyabama-Coding-Club/HacktoberFest2019/blob/master/CONTRIBUTING.md) and add a basic description of the algorithm and a link to more documentation in the README.md inside the folder? 

This will help with giving others a better understanding of the concept.

peakercope commented:
sure. will add now

peakercope committed:
binary_search.js Added Binary Search Algorithm
This adds Binary Search Algorithm which finds the position of a target value within a sorted array.
Closes: https://github.com/Sathyabama-Coding-Club/HacktoberFest2019/issues/15
